packages/core/src/app/payment/paymentMethod/PaypalCommerceCreditDescription.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FunctionComponent, useEffect } from 'react';
+
+import { useCheckout } from '@bigcommerce/checkout/payment-integration-api';
+
+import PaymentMethodId from './PaymentMethodId';
+
+const PaypalCommerceCreditDescription: FunctionComponent<{ onUnhandledError?(error: Error): void }> = ({ onUnhandledError }) => {
+    const { checkoutService } = useCheckout();
+
+    useEffect(() => {
+        try {
+            checkoutService.initializePayment({
+                methodId: PaymentMethodId.PaypalCommerceCredit,
+                paypalcommercecredit: {
+                    // @ts-ignore
+                    bannerContainerId: 'paypal-commerce-banner-container',
+                },
+            });
+
+            checkoutService.deinitializePayment({
+                methodId: PaymentMethodId.PaypalCommerceCredit,
+            });
+        } catch (error) {
+            if (error instanceof Error) {
+                onUnhandledError?.(error);
+            }
+        }
+
+        return () => {
+            try {
+                checkoutService.deinitializePayment({
```

Comment:
Why do you deinitialize method on component unmount if you made it right after initialization?

Reply:
@serhii-tkachenko I removed that part of the code and successfully retested. Take a look pls